x-pack/plugin/esql/qa/testFixtures/src/main/resources/mv_intersection.csv-spec

```diff
@@ -0,0 +1,179 @@
+testMvIntersectionWithIntValues
+required_capability: fn_mv_intersection
+
+// tag::testMvIntersectionWithIntValues[]
+ROW a = [1, 2, 3, 4, 5], b = [2, 3, 4, 5, 6]
```

Comment:
can we get a test where one of the fields is not multi valued?
it should still work I _think_:
```
ROW a = [1, 2, 3, 4, 5], b = 2
| EVAL finalValue = MV_INTERSECTION(a, b) 
| KEEP finalValue
"
```